channels/views_test.py

```diff
@@ -722,6 +731,18 @@ def test_add_moderator(client, use_betamax, praw_settings, staff_jwt_header):
     assert resp.json() == {'moderator_name': 'othermoderator'}
 
 
+def test_add_subscriber(client, use_betamax, praw_settings, staff_jwt_header):
+    """
+    Adds a subscriber to a channel
+    """
+    client.force_login(UserFactory.create(username='fooadmin'))
+    subscriber = UserFactory.create(username='othersubscriber')
+    url = reverse('subscriber-list', kwargs={'channel_name': 'test_channel'})
+    resp = client.post(url, data={'subscriber_name': subscriber.username}, format='json', **staff_jwt_header)
+    assert resp.status_code == status.HTTP_201_CREATED
+    assert resp.json() == {'subscriber_name': 'othersubscriber'}
```

Comment:
Could you add a test to verify that this fails for a nonstaff user trying to modify a second user? Same for the remove.

Reply:
I am not sure why I should do this: isn't that tested for the `JwtIsStaffOrReadonlyPermission` permission class?

Reply:
Yeah, that class is tested, my thought was we'd want to know if that breaks against an api for whatever reason. That could break independently of the behavior of `JwtIsStaffOrReadonlyPermission`. I guess we don't test it elsewhere so i'm fine leaving it.